### ISSUE TITLE
JDownloader

### ISSUE THREAD

**Issue by xazac** [elrumo/macOS_Big_Sur_icons_replacements#1028]
### Name of app
JDownloader
### Website of the app requested
https://jdownloader.org

**Comment by KnowsCount** [comment]
well.. this is not a helpful answer but I think the Big Sur icon is going to be out soon.

https://board.jdownloader.org/showthread.php?t=86040

Let's hope it's not as plain-looking as its official forum..

**Comment by KnowsCount** [comment]
hold on.. [octaviotti](https://www.deviantart.com/octaviotti) has already done it for us!

https://www.dropbox.com/sh/6dtjilr361gvsep/AAAnkyOFYJ7uLstl3n-B-NdVa/PNG?dl=0&lst=&preview=jDownloader.png&subfolder_nav_tracking=1

**Comment by KnowsCount** [comment]
You could always close if this satisfies you or reply if there is anything wrong. @xazac

**Comment by xazac** [comment]
No, issue is not solved because when I launch JDownloader from Dock it reverts to old (ugly) icon.

**Comment by the3ye** [comment]
> No, issue is not solved because when I launch JDownloader from Dock it reverts to old (ugly) icon.

You have to change jd_logo_128_128.png in `~/JDownloader 2/themes/standard/org/jdownloader/images/logo` to replace standard icon. If you still see old icon for a second when opening the app, you have to replace app.icns in `~/JDownloader 2/JDownloader2.app/Contents/Resources` as well
